tests/agent/test_agent_run_submission.py

```diff
@@ -282,8 +284,12 @@ async def test_agent_creates_work_queue_if_doesnt_exist_in_work_pool(
 
 
 async def test_agent_does_not_create_work_queues_if_matching_with_prefix(
-    session, prefect_caplog
+    session, prefect_caplog, prefect_client: PrefectClient
 ):
+    await prefect_client.create_work_pool(
+        WorkPoolCreate(name=DEFAULT_AGENT_WORK_POOL_NAME, type="prefect-agent")
+    )
```

Comment:
I think this might be unnecessary with the fixture you added in `tests/cli/test_start_agent.py`

Reply:
I think `pytest` scopes that fixture only to that file and tests here wouldn't get to autouse it. We could explore options for moving the fixture somewhere else so that every test automatically picks it up?

Reply:
Ah true. I think we'd have to move it into `conftest.py` to make it available everywhere, but I don't think that's necessary.